.storybook/main.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
   stories: [
-    '../tests/**/*.stories.tsx'
+    '../tests/**/*.stories.ts?(x)'
```

Comment:
Do we need this change? I think our stories should always be TSX files, since they're using React Components.